### DISCUSSION TITLE
What's the association with FHIR?

### DISCUSSION THREAD

**Discussion by mikebouck** [davebronson/clear-hl7-net#6] [General]
Great project!  I'm curious, though - what's the association with FHIR?  This seems to be an HL7 v2 library....

**Comment by davebronson** [comment]
Hi Mike.  Thanks for the compliment!  You're correct that this is an HL7 v2 library, and I've been considering for some time the best way to unwind the 'fhir' term from everything with a minimum of disruption.  No way to do it without introducing breaking changes unfortunately, so I'm just going to dive into it.

**Reply by mikebouck** [reply]
That was fast!  Technically I guess FHIR does have a dotted-line relationship with HL7 V2/V3 (e.g. supporting legacy data types, etc.) however I think FHIR is mostly its own thing.

**Reply by davebronson** [reply]
True.  As you browse through the official HL7 content online, it's very common to find FHIR displayed prominently in the top banner.  It's easy to perceive things as being more intertwined than they actually are.

Fortunately (for me) it was just a matter of doing a mass search/replace on the codebase and then run through my normal set of tests to ensure that the output hasn't changed.  I just needed the little nudge to move forward with it (thanks!).
